platform/security/core/security-core-services/src/main/java/ddf/security/listener/AuditingHttpSessionListener.java

```diff
@@ -1,48 +0,0 @@
-/**
- * Copyright (c) Codice Foundation
- *
- * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
- * Lesser General Public License as published by the Free Software Foundation, either version 3 of
- * the License, or any later version.
- *
- * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
- * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
- * License is distributed along with this program and can be found at
- * <http://www.gnu.org/licenses/lgpl.html>.
- */
-package ddf.security.listener;
-
-import com.google.common.hash.Hashing;
-import ddf.security.audit.SecurityLogger;
-import java.nio.charset.StandardCharsets;
-import javax.servlet.http.HttpSession;
-import javax.servlet.http.HttpSessionEvent;
-import javax.servlet.http.HttpSessionListener;
-
-public class AuditingHttpSessionListener implements HttpSessionListener {
```

Comment:
Registering this as a service doesn't work because of a limitation in Pax Web. See https://ops4j1.jira.com/browse/PAXWEB-1123. If we wanted to use an HttpSessionListener for all servlet contexts, we'd have to inject it the same way we do error pages, which is a bit of a hack. So I opted to audit from the AttributeSharingHashSessionIdManager